### DISCUSSION TITLE
"Unfollow user" does not work

### DISCUSSION THREAD

**Discussion by DarkWanderer** [community/community#75883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

A few months ago, I have accidentally "followed" a user which has followed me. Right after that, I unfollowed him - however, his activity still shows up in the feed. Every time I click "..." on the item in the feed, it shows an option to "unfollow" - but it never does anything meaningful. There seems to be a bug with user following mechanic somewhere.

EDIT: following and unfollowing user again worked this time - but not in a few times I tried it before

**Comment by f-alizada** [comment]
Same here, accidently followed the user, and immidiately unfollowed, however still receiving the updates on users activity.

**Comment by vipunsanjana** [comment]
refresh page and check internet connection
